### PR TITLE
Add AnnouncementBanner component to display sitewide announcements

### PR DESCRIPTION
**Pull request by tituomin**
Screenshot from 2023-12-21 18-19-07

### CODE REVIEW

File: components/common/AnnouncementBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+import { Container } from 'reactstrap';
+
+export const AnnouncementBannerSection = styled.div`
```
Comment (woodwoerk) [inline components/common/AnnouncementBanner.tsx:4]: Nitpick: Juha and I agreed to started prefixing styled component names with `Styled`, i.e. `StyledAnnouncementBannerSection`. It makes names a bit verbose but is really handy to easily spot "real" components and styled components in the render function

File: components/common/AnnouncementBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  }
+  p ~ p {
+    margin-top: ${(props) => props.theme.spaces.s050};
+  }
+  a {
+    color: ${(props) => props.theme.themeColors.white};
+    font-weight: bold;
+  }
+  @media (max-width: ${(props) => props.theme.breakpointMd}) {
+  }
```
Comment (woodwoerk) [inline components/common/AnnouncementBanner.tsx:28]: Unnecessary breakpoint?

**Review by woodwoerk — APPROVED** [review]
Nice! Looking good, added a few nitpicks but nothing that prevents merging – we can decide upon the themed background colour later as more customers take this into use

File: components/common/AnnouncementBanner.tsx (hunk 2 above)
Comment (tituomin) [reply components/common/AnnouncementBanner.tsx]: https://github.com/kausaltech/kausal-watch-ui/pull/243/commits/343691cd199f6f6e02ac8f28526d0760fb084fc5

File: components/common/AnnouncementBanner.tsx (hunk 1 above)
Comment (tituomin) [reply components/common/AnnouncementBanner.tsx]: https://github.com/kausaltech/kausal-watch-ui/pull/243/commits/343691cd199f6f6e02ac8f28526d0760fb084fc5

**Pull request merged**